app/javascript/player/index.jsx

```diff
@@ -57,6 +57,13 @@ class Player extends React.Component<Props, { playing: boolean }> {
     this.audioElement = document.createElement('audio')
   }
 
+  componentDidMount () {
+    const Url = new URL(window.location.href)
```

Comment:
- please name variables with in `snakeCase` (so this should be `url`)
- additionally, you can just pass `window.location` to the `URL` constructor